ResonatorNetwork.sc

```diff
@@ -92,11 +92,12 @@ ResonatorNetwork {
 		}
 	}
 
-	calcModalData { arg minFreq=25,maxFreq=(Server.default.sampleRate ? 44100).div(2),minT60=0.01,gain=1,pathname=pythonScriptPath ++ "/modalData.json",incl="ynnn",async=false;
+	calcModalData { arg minFreq=25,maxFreq=22050,minT60=0.01,gain=1,pathname=pythonScriptPath ++ "/modalData.json",incl="ynnn",async=false;
```

Comment:
Could you elaborate on why you decided to change `(Server.default.sampleRate ? 44100).div(2)` to just 22050? Granted, the fact that `Server.default` doesn't necessarily correspond to the server that is used to run the simulation and the fact that the default server has to be booted doesn't make it and ideal solution, but I still think it is better than just assuming that the sample rate is fixed to 44100.

Reply:
Mostly because what you say at the end and because I'm not sure if it's good idea to use expressions for args default values. It can be reverted, will work fine.